clusterloader2/pkg/test/simple_test_executor.go

```diff
@@ -318,6 +320,52 @@ func (ste *simpleTestExecutor) ExecuteObject(ctx Context, object *api.Object, na
 	return errList
 }
 
+// initNamespaceParamters initializes namespace parameters
+func initNamespaceParameters(ctx Context, conf *api.Config) *errors.ErrorList {
+
+	// For backwarad compatibility with deprecated "automananaagedNamespaces" in testconfig file.
+	// Remove in the future versions
+	if conf.Namespace.Count == nil && conf.AutomanagedNamespaces != nil {
+		conf.Namespace.Count = conf.AutomanagedNamespaces
+	}
+
+	var defaultNamespaces int32 = 1
+	if conf.Namespace.Count == nil {
+		conf.Namespace.Count = &defaultNamespaces
+	}
+	if *conf.Namespace.Count <= 0 {
+		return errors.NewErrorList(fmt.Errorf("number of namespaecs: %d was less than 1", *conf.Namespace.Count))
+	}
+
+	// For backwarad compatibility with deprecated options "delete-stale-namespaces" and "delete-automananged-namespaces".
+	// Remove in the future versions
```

Comment:
We don't have any formal deprecation policy, so let's remove it (it's not perfect, but it will be much cleaner).

Reply:
done,  new implementation in api.default.go and api.validate.go as suggested.